src/pages/home-page/HomePage.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
You don't need to import react here

Reply:
If I remove this, then I see this warning:

```
src/pages/home-page/HomePage.js
  Line 4:3:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 5:5:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 6:7:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 7:7:  'React' must be in scope when using JSX  react/react-in-jsx-scope
```

Reply:
That's because of the eslint rule that @kevin-rodriguez already suggest deleting from `.eslintrc`

Reply:
Oooh you're right